monero-outreach-docs/translations/es/archive/triptych_es.md

```diff
@@ -1,43 +1,43 @@
-# Triptych: Un nuevo algoritmo que protege a los usuarios de Monero
+# Tríptico: Un nuevo algoritmo que protege a los usuarios de Monero
 
-**“Monero oculta al remitente, al destinatario y el monto en una transacción. Triptych (pronunciado TRIP-tick) es una actualización para Monero mas rápida, liviana y mucho mas privada.” - 1st of September, 2020**
+**“Monero oculta al remitente, al destinatario y el monto en una transacción. Tríptico (Triptych - pronunciado TRIP-tick) es una actualización para Monero mas rápida, liviana y mucho mas privada.” - 1st de Septiembre, 2020**
 
 Monero oculta el emisor, el receptor y el monto en una transacción, con los detalles detrás de este comportamiento rico y en evolución. Este artículo cubre Tríptico (Triptych - Pronunciado TRIP-tick), una nueva tecnología de Monero que esconde mejor a los remitentes, utilizando menos espacio en la cadena de bloques de Monero y menos tiempo en el procesamiento. 
 
 ### _Firma de Círculos_
 
-Ocultar a los remitentes en Monero toma más que solo proteger direcciones. Todo Monero está representado en la cadena de bloques como salidas de transacciones, llamadas TXOs. Cada TXO es etiquetado usando un ID único (una sola vez) llamado clave pública (también es llamado una dirección secreta). Con sólo ver en la cadena de bloques de Monero, las claves públicas de una TXO absolutamente no se pueden conectar a las direcciones de Monero. Sin embargo, las claves públicas por si mismas también necesitan protección. Sería inaceptable que el flujo de Monero entre las claves públicas fuera transparente, considerando que las técnicas fuera de la cadena de bloques podrían conectar esas claves públicas con las personas.
+Ocultar a los remitentes en Monero es algo más complicado que solo proteger direcciones. Todo Monero está representado en la cadena de bloques como salidas de transacciones, llamadas TXOs. Cada TXO es etiquetado usando un ID único (una sola vez) llamado clave pública (también es llamado una dirección secreta). Con sólo ver en la cadena de bloques de Monero, las claves públicas de una TXO absolutamente no se pueden conectar a las direcciones de Monero. Sin embargo, las claves públicas por si mismas también necesitan protección. Sería inaceptable que el flujo de Monero entre las claves públicas fuera transparente, considerando que las técnicas fuera de la cadena de bloques podrían conectar esas claves públicas con las personas.
```

Comment:
"Ocultar a los emisores en Monero es algo más..." Done


"Con sólo ver en la cadena de bloques de Monero, las claves TXO públicas de ninguna manera se pueden conectar a las direcciones de Monero."   No podemos decir **las claves TXO** ya que TXO no es una clave, es un TXOutput.    **Seria las claves publicas  de un TXO de ninguna manera**

Reply:
Everything is an output. The Monero blockchain is made out of outputs. If you read the paragraph carefully it states that TXOs are public keys, meaning they are keys, "claves". 

"Each **TXO** is labeled using a unique (one time) ID **called the public key** (also called a stealth address)."

Here is the full paragraph:
"Hiding senders in Monero takes more than just guarding addresses. All Monero is represented on the blockchain as outputs of transactions, called TXOs. **Each TXO is labeled using a unique (one time) ID called the public key (also called a stealth address).** By just viewing the Monero blockchain, TXO public keys absolutely cannot be connected to Monero addresses. Yet public keys themselves also need protection. It would be unacceptable for the flow of Monero among public keys to be transparent considering that techniques outside the blockchain could connect those public keys to people."

Reply:
The next sentence is referring to TXOs as public keys. 

"By just viewing the Monero blockchain, **TXO public keys** absolutely cannot be connected to Monero addresses."

Reply:
Ok, Done